addons/sale_management/models/sale_order_option.py

```diff
@@ -115,6 +115,7 @@ def _get_values_to_add_to_order(self):
             'product_uom_qty': self.quantity,
             'product_uom': self.uom_id.id,
             'discount': self.discount,
+            'sequence': max(self.order_id.order_line.mapped('sequence'), default=0) + 1
```

Comment:
> Could you also shorten your commit message (72 characters max)?

:pray:

Reply:
oh sorry i thought you meant the whole message so i changed just the message
ok i changed the heading

Reply:
Ah yes, I should have been clearer, sorry. Yes I meant title.